docs/en/administration/management/BE_blacklist.md

```diff
@@ -2,24 +2,26 @@
 displayed_sidebar: docs
 ---
 
-# Manage BE Blacklist
+# Manage BE and CN Blacklist
 
-This topic describes how to manage the BE Blacklist.
+This topic describes how to manage the BE and CN Blacklist.
 
 From v3.3.0 onwards, StarRocks supports the BE Blacklist feature, which allows you to forbid the usage of certain BE nodes in query execution, thereby avoiding frequent query failures or other unexpected behaviors caused by the failed connections to the BE nodes.
```

Comment:
What is a common cause for failed connections to certain BE nodes? If a temporary network issue is a good example, then let's add:
```suggestion
From v3.3.0 onwards, StarRocks supports the BE Blacklist feature, which allows you to forbid the usage of certain BE nodes in query execution, thereby avoiding frequent query failures or other unexpected behaviors caused by the failed connections to the BE nodes. A network issue preventing connections to one or more BEs would be an example of when to use the blacklist.
```